waiter/resources/web/sim.html

```diff
@@ -4,119 +4,200 @@
-				<label>max-instances <input id="max-instances" value="500" type="number"> </label>
-				<label>scale up factor <input id="scale-up-factor" value="0.01" type="number" step="0.001"> </label>
-				<label>scale down factor <input id="scale-down-factor" value="0.001" type="number" step="0.001"> </label>
-				<label>jitter threshold <input id="jitter-threshold" value="0.5" type="number" step="0.01"> </label>
+				<label>cpus <input id="cpus" value="2" type="number"/></label>
+				<label>memory (in MB) <input id="mem" value="512" type="number"/></label>
+				<label>startup time (secs) <input id="startup-ticks" value="30" type="number"/></label>
+				<label>min-instances <input id="min-instances" value="1" type="number"/></label>
+				<label>max-instances <input id="max-instances" value="500" type="number"/></label>
+				<label>scale up factor <input id="scale-up-factor" value="0.01" type="number" step="0.001"/></label>
+				<label>scale down factor <input id="scale-down-factor" value="0.001" type="number" step="0.001"/></label>
+				<label>jitter threshold <input id="jitter-threshold" value="0.5" type="number" step="0.01"/></label>
 			</div>
-			<div class="config-box">
+			<div class="config-box left-align">
 				<h4>Scaler Settings</h4>
-				<label>scale every (secs) <input id="scale-ticks" value="5" type="number"> </label>
-				<label>simulation length (secs) <input id="total-ticks" value="3600" type="number"> </label>
+				<label>scale every (secs) <input id="scale-ticks" value="5" type="number"/></label>
+				<label>simulation length (secs) <input id="total-ticks" value="3600" type="number"/></label>
+			</div>
+        </div>
+        <div id="summary">
+			<div id="simulation-summary" class="config-box right-align">
+				<h4>Simulation Summary</h4>
+				<label>total queue time &nbsp; <input id="res-total-queue-time" value="0" type="text" readonly/></label>
+				<label>total idle server time &nbsp; <input id="res-total-idle-server-time" value="0" type="text" readonly/></label>
+				<label>average utilization &nbsp; <input id="res-average-utilization" value="0" type="text" readonly/></label>
+				<label>average waste &nbsp; <input id="res-average-waste" value="0" type="text" readonly/></label>
 			</div>
 			<div class="clearfix"></div>
 		</div>
 		<button id="sim-button">Sim</button>
+		<button id="image-button"
+				title="Downloads the chart as a png image excluding the legend and axis values">Download Image</button>
 		<div id="chart"></div>
-		<label>
+		<div>
 			<div id="metrics-controls"></div>
-		</label>
+		</div>
 	</body>
 
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/flot/0.8.3/jquery.flot.min.js"></script>
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/flot/0.8.3/jquery.flot.time.min.js"></script>
     <script>
 
+const numberWithCommas = (x) => {
```

Comment:
We're not intending to export these functions (`numberWithCommas` and `showTooltip`) so I like them inside the (function() {...})